rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/SchedulerStateRest.java

```diff
@@ -588,7 +596,7 @@ public boolean deleteLiveLogJob(@HeaderParam("sessionid") String sessionId, @Pat
     public JobResultData jobResult(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobId)
             throws NotConnectedRestException, PermissionRestException, UnknownJobRestException {
         try {
-            Scheduler s = checkAccess(sessionId, "jobs/" + jobId + "/result");
+            Scheduler s = checkAccess(sessionId, PATH_JOBS + jobId + "/result");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "/result" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)